packages/allow-scripts/test/projects/.gitignore

```diff
@@ -1,2 +1,4 @@
 !node_modules
 1/package.json
+.bin
+.goodscriptworked
```

Comment:
```suggestion
.goodscriptworked

```

Reply:
...not sure about that suggestion. Either way; missing newline at EOF.

Reply:
fixed